src/witan/models/run_models.clj

```diff
@@ -102,10 +103,26 @@
       (data-csv/write-csv out-file
                           (convert-row-maps-to-vector-of-vectors ordered-data)))))
 
+(defn get-district [gss-code]
```

Comment:
The lookup would then be 
```Clojure
(get local-authorities (keyword gss-code))
```
We could easily use it to print the district name to the terminal at the start of the programme (after reading the user inputs). That's something Bruce suggested so we're sure we're running the CCM for the right district before seeing the output csv.